backend/app/__init__.py

```diff
@@ -0,0 +1,15 @@
+import os
+from flask import Flask
+from flask_cors import CORS
+from .views.auth import auth
```

Comment:
Maybe make this backend.app.views.auth better convention imo. It's what you've done with most of the other imports